packages/elements/src/photon-multirx-form/photon-prescribe-workflow.tsx

```diff
@@ -348,7 +348,7 @@ export function PrescribeWorkflow(props: PrescribeProps) {
           effectiveDate: draft.effectiveDate,
           instructions: draft.instructions,
           notes: draft.notes,
-          patientId: props.formStore.patient?.value.id,
+          patientId: props?.patientId || props.formStore.patient?.value.id,
```

Comment:
nit: wouldn't it be more syntaxically correct to do `??` since we're trying to coalesce and not do a logical or?

Reply:
yeah, it's more syntactically correct, I'll switch to that